src/NumericalAlgorithms/SphericalHarmonics/Python/Strahlkorper.cpp

```diff
@@ -74,6 +69,12 @@ void bind_strahlkorper(pybind11::module& m) {  // NOLINT
         return std::make_pair(legend, data);
       },
       py::arg("strahlkorper"), py::arg("time"), py::arg("max_l"));
+}
+}  // namespace
+
+void bind_strahlkorper(py::module& m) {
+  bind_strahlkorper_impl<Frame::Inertial>(m);
+  bind_strahlkorper_impl<Frame::Grid>(m);
```

Comment:
While you're at it, just do `Distorted` and `NoFrame` as well. And also add it to the other instantiations.

Reply:
Are those useful for something?

Reply:
Ok added distorted